specifications/api/swagger.yaml

```diff
@@ -321,6 +321,62 @@ x-poolMetadataSource: &poolMetadataSource
       format: uri
       example: https://smash.cardano-mainnet.iohk.io/
 
+x-credentialPubKey: &credentialPubKey
+  description: A public key (public key without chain code) for credential - 32 bytes
+  type: string
+  format: bech32
+  pattern: "^((addr_vk)|(stake_vk))1[0-9a-z]*$"
+  example:
+    - stake_vk16apaenn9ut6s40lcw3l8v68xawlrlq20z2966uzcx8jmv2q9uy7qau558d
+    - addr_vk16apaenn9ut6s40lcw3l8v68xawlrlq20z2966uzcx8jmv2q9uy7q3yvuv2
+
+x-anyAddress: &anyAddress
+  description: A Shelley address representing either enterprise, reward account or delegating address
+  type: string
+  format: bech32
+  pattern: "^((addr)|(stake)|(addr_test)|(stake_test))1[0-9a-z]*$"
+  example:
+    - stake17xt2z3pa7etaxp7jurdg0m8jhsmtp4r2z56pd3a5q3jhxycdxzmx9
+    - addr1wy5np0m5x03tax3kcdh6e2cet98qcfs80wtv4cyvl5taclc6dnd8e
+    - addr1xy5np0m5x03tax3kcdh6e2cet98qcfs80wtv4cyvl5tacluk59zrmajh6vra9cx6slk090pkkr2x59f5zmrmgpr9wvfs37hjk4
+
+x-credential: &credential
+  nullable: false
+  additionalProperties:
+    $ref: "#/components/schemas/CredentialValue"
+
+x-script: &script
+  description: |
+    Script allows establishing either spending or staking guard that could be removed
+    when a special combination of signing keys is present.
+
+    The script can be constructed using four primitives:
+     - verification key hash for which a corresponding signing key is expected
+     - all condition for which all list elements' expectation are met for the condition to be valid
+     - any condition for which any list element' expectation is to be met for the condition to be valid
+     - some condition for which a minimal number of list elements' expectation is to be met for the condition to be valid
+    example:
```

Comment:
I think this line needs to be de-indented so that it's not included in the description string.

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/2253/commits/595999169a24d1637760f2f14f02c4af9d3cc4ad